.devcontainer/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM ubuntu:bionic
 
+ARG TARGETARCH
+
+RUN if [ "${TARGETARCH}" != "amd64" ] && [ "${TARGETARCH}" != "arm64" ]; then \
+    echo "${TARGETARCH} is not a supported architecture. Please use linux/amd64 or linux/arm64"; \
+    exit 1; fi
```

Comment:
I don't love this but realized while testing that waiting until the later build stages to discover a particular platform is unsupported is annoying. I couldn't find a built in mechanism to allowlist architectures but happy to hear better ideas.